concent_api/utils/helpers.py

```diff
@@ -2,14 +2,27 @@
 import base64
 import binascii
 import datetime
+import os
 import time
 
+from django.conf                    import settings
+from django.core.checks             import Error
 from django.utils                   import timezone
 
 from golem_messages                 import message
 from golem_messages.datastructures  import FrozenDict
 
 
+def storage_cluster_certificate_path_check(app_configs = None, **kwargs):
+    certificate_path = settings.STORAGE_CLUSTER_SSL_CERTIFICATE_PATH
+    if certificate_path != '':
+        if not os.path.exists(certificate_path):
```

Comment:
I think it would be a good idea to check for `None` here. Trying to disable something by setting it to `None` is pretty common. The current code will raise `TypeError`. Saying that `None` is not a valid value would be a better user experience (the admin is the user here) at a pretty insignificant cost of an extra `if`.

Reply:
done

Reply:
done